src/appengine/libs/issue_management/github.py

```diff
@@ -0,0 +1,177 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Helper functions to file GitHub issues."""
+
+import github
+
+from clusterfuzz._internal.config import db_config
+from clusterfuzz._internal.datastore import data_handler
+from clusterfuzz._internal.metrics import logs
+
+TESTCASE_REPORT_URL = 'https://{domain}/testcase?key={testcase_id}'
+
+MONORAIL_URL = (
+    "https://bugs.chromium.org/p/oss-fuzz/detail?id={bug_information}")
+OSS_FUZZ_ISSUE_URL = "https://github.com/google/oss-fuzz/issues/new"
+
+ISSUE_TITTLE_TEXT = "OSS-Fuzz issue {bug_information}"
+
+ISSUE_CONTENT_TEXT = ("OSS-Fuzz has found a bug in this project. Please see "
+                      f"{TESTCASE_REPORT_URL}"
+                      "for details and reproducers."
+                      "\n\n"
+                      "This issue is mirrored from "
+                      f"{MONORAIL_URL} "
+                      "and will auto-close if the status changes there."
+                      "\n\n"
+                      "If you have trouble accessing this report, "
+                      f"please file an issue at {OSS_FUZZ_ISSUE_URL}."
+                      "\n")
+
+ISSUE_ClOSE_COMMENT_TEXT = ("OSS-Fuzz has closed this bug. Please see "
+                            f"{MONORAIL_URL} "
+                            "for details.")
+
+
+def get_issue_title(testcase):
+  """Generate the title of the issue"""
+  return ISSUE_TITTLE_TEXT.format(bug_information=testcase.bug_information)
+
+
+def get_issue_body(testcase):
+  """Generate the body of the issue"""
+  return ISSUE_CONTENT_TEXT.format(
+      domain=data_handler.get_domain(),
+      testcase_id=testcase.key.id,
+      bug_information=testcase.bug_information)
+
+
+def get_issue_close_comment(testcase):
+  """Generate the closing comment of the issue"""
+  return ISSUE_ClOSE_COMMENT_TEXT.format(
+      bug_information=testcase.bug_information)
+
+
+def get_access():
```

Comment:
nit: Make this a "private" function by naming it "_get_access_token"). Also add the "token" part explicitly to make it more clear.